src/core/Navigation/WizardNavigation/WizardNavigation/WizardNavigation.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ReactNode, useState } from 'react';
+import { HtmlButton, HtmlDiv, HtmlNav, HtmlUl } from '../../../../reset';
+import styled from 'styled-components';
+import { SuomifiThemeConsumer } from '../../../theme';
+import { baseStyles } from './WizardNavigation.baseStyles';
+import classnames from 'classnames';
+import { Icon } from '../../../Icon/Icon';
+
+export interface WizardNavigationProps {
+  children: ReactNode | ReactNode[];
```

Comment:
Is the array option really needed? ReactNode should support ReactFragment and an array or ReactNodes by default.

Reply:
I didn't realise ReactNode actually accepts array by default. Removed the redundant part.